debugger/src/debugger/session_data.rs

```diff
@@ -163,6 +163,24 @@ impl SessionData {
                 .into());
             };
 
+            debug_info.set_minimum_instruction_size(
+                match target_session
+                    .core(core_configuration.core_index)?
+                    .instruction_set()?
+                {
+                    probe_rs::InstructionSet::Thumb2 => {
```

Comment:
Maybe this is worth adding as `fn get_min_instruction_size()` on InstructionSet so that it's closer to the ISA definition?

Reply:
Done ... soooo much cleaner. Thanks :)